### PR TITLE
Add missing changeset for 2077

### PR DESCRIPTION
**Pull request by libertymayc**
Add missing changeset for #2077

### CODE REVIEW

**Review by lukeac123 — APPROVED** [review]

File: .changeset/neat-mice-try.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/ag-grid-theme": patch
+---
+
+Fixed status bar and floating filter styles
+Added `compact` parameter to `useAgGridHelpers` to turn on HD compact density styling when set to true
```
Comment (origami-z) [inline .changeset/neat-mice-try.md:6]: Is `useAgGridHelpers` something can be used by consumer?

Comment (ZarkoPernar) [reply .changeset/neat-mice-try.md:6]: No, but I think we were expecting them to copy/paste it iirc

Comment (libertymayc) [reply .changeset/neat-mice-try.md:6]: yeah not ideal, will reword

File: .changeset/neat-mice-try.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/ag-grid-theme": patch
+---
+
+Fixed status bar and floating filter styles
+Added HD compact density styling
```
Comment (libertymayc) [inline .changeset/neat-mice-try.md:6]:
```suggestion
Fixed status bar and floating filter styles

Added HD compact density styling
```

**Review by ZarkoPernar — APPROVED** [review]

**Pull request merged**